isabelle/CodeExports/CodeExports.thy

```diff
@@ -66,6 +66,7 @@ export_code
   getSignatures
   maxTimeContract
   calculateNonAmbiguousInterval
+  validAndPositive_state
```

Comment:
I had to add validAndPositive_state to the exports to generate valid state, if not the serialization failed as duplicate inputs are omitted. Eventually we could include the Semi-arbitrary instances to solve the same issue